tests/models/test_restore.py

```diff
@@ -622,14 +622,14 @@ def on_pretrain_routine_end(self):
 
             dataloader = dm.train_dataloader()
             tpipes.run_model_prediction(self.trainer.lightning_module, dataloader=dataloader)
-            self.on_pretrain_routine_end_called = True
+            self.on_train_start_called = True
 
     # new model
     model = CustomModel()
 
     # fit new model which should load hpc weights
     new_trainer.fit(model, datamodule=dm)
-    assert model.on_pretrain_routine_end_called
+    assert model.on_train_start_called
```

Comment:
The test case here is failing. Any idea?

Reply:
I'm having a hard time. I don't really understand what this test is trying to do

Reply:
it looks like these tests predated `trainer.validate` so there's no need to write it this way anymore